api/src/pcapi/core/offerers/api.py

```diff
@@ -2679,6 +2679,45 @@ def get_or_create_offerer_address(offerer_id: int, address_id: int, label: str |
     return offerer_address
 
 
+def try_create_api_address(
```

Comment:
Je lui aurais donné un autre nom, plutôt 
```suggestion
def create_offerer_address_from_address_api(...)
```